### PR TITLE
Added comments

### PR DESCRIPTION
**Pull request by daniel-fitzg**
Comment blocks added to all files. Let me know if you're not happy with them @dshirt @rdmoraes 

@dshirt I removed the static and final keywords from the declarations inside the interface. They are redundant as within an interface the methods are implicitly final and static.

@rdmoraes I just changed the access modifiers in Staff class to package-private.

### CODE REVIEW

File: src/main/java/employee/com/BasicPlusCommissionBasedStaff.java [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
+/*
+ * Models a staff employee that receives a basic wage and commission
+ *
+ * Contributors:
+ * Renan Moraes
+ * John Lawless
+ * Daniel Fitzgerald
+ *
+ * April/May 2020
+ * */
```
Comment (dshirt) [inline src/main/java/employee/com/BasicPlusCommissionBasedStaff.java]: May need a contact email Dan. I'm not sure in this case, but for the exams was looking for name, contact, date

**Review by dshirt — APPROVED** [review]
Just don't know if there should be a contact email in the header

File: src/main/java/employee/com/BasicPlusCommissionBasedStaff.java (hunk 1 above)
Comment (daniel-fitzg) [reply src/main/java/employee/com/BasicPlusCommissionBasedStaff.java]: I'll add our student emails to it.

**Review by rdmoraes — APPROVED** [review]
LGTM

**Pull request merged**